### ISSUE TITLE
Minor Bug/Feature request: Play audio automatically shows new word while playing audio for old word

### ISSUE THREAD

**Issue by jeff8v7** [davidsansome/tsurukame#404]
This is a small thing, but might improve retention.

When the "Play audio automatically" setting is ON, the audio for a word plays automatically when you answer correctly.

This is great, however, unfortunately the screen switches to display the new word _before_ playing the audio rather than _after_. 

So while listening to the audio for one word, I'm looking at the kanji for a different word.

Would (presumably) be better for retention to look at the kanji for the word that is being listened to.

**Comment by jeff8v7** [comment]
Side note - would also be nice to have setting for "Repeat audio N times" which would allow the user to choose how many times to play the audio after a correct guess (while looking at the Kanji for the audio). Brains love repetition, so playing the audio 2 or 3 times might improve retention for some people.

**Comment by UInt2048** [comment]
@jeff8v7 Good idea - might I suggest simply a setting for disabling lightning mode (so you have to choose to move on with this on)

You don't want the reading audio playing while it's asking you for the reading - then you're not learning to remember it...

**Comment by jeff8v7** [comment]
I like lightening mode. This is an unrelated request. Let me clarify...

Currently, the behavior follows this flow:
1. Show Kanji (ie, 順番)
2. User types correct reading (じゅんばん)
4. Show next kanji automatically (急ぐ)
3. Play reading audio じゅんばん (for correctly answered item)

Since #4 happens before #3, the user is looking at 急ぐ while hearing じゅんばん, which does not help with retention.  
Looks like a minor bug to me.

I'm suggesting #3 should happen before #4:

1. Show Kanji (ie, 順番)
2. User types correct reading (じゅんばん)
3. Play reading audio じゅんばん (for correctly answered item)
4. Show next kanji automatically (急ぐ)

Thus, the user is looking at 順番 while hearing じゅんばん (which they just typed).

> You don't want the reading audio playing while it's asking you for the reading

Yes, of course, you would not put #3 before #2, because that wouldn't make sense :-)

**Comment by jeff8v7** [comment]
An additional small enhancement would be to automatically play the reading audio after a correct meaning answer **if and only if** the reading was correctly answered previously. Like so...

1. Show item A (ie, 順番)
2. User types correct reading A (じゅんばん)
3. Play reading A audio (じゅんばん) while displaying item A
4. Show next item B automatically ...

(more items are tested...)

1. Show item A (ie, 順番)
2. User types correct **_meaning_** A (order)
3. Play reading A audio (じゅんばん) while displaying item A

No harm done, since they've answered both reading and meaning correctly, and it reinforces the audio.

**Comment by UInt2048** [comment]
@jeff8v7 Can you not use the number sign (#) to do that? You're referencing unrelated issues...
